frontend/env_config/.env.test

```diff
@@ -1,4 +1,4 @@
 VITE_APP_TITLE=HN Web (Test)
 VITE_KEYCLOAK_URL=https://common-logon-dev.hlth.gov.bc.ca/auth/
 VITE_SITEMINDER_LOGOUT=https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi?retnow=1&returl=https://user-management-dev.hlth.gov.bc.ca
-VITE_SERVICE_URL=
\ No newline at end of file
+VITE_SERVICE_URL=http://hnweb-dev-api.apps.silver.devops.gov.bc.ca/
```

Comment:
Should our test environment be using test keycloak and test namespace?
Or do we have a mismatch like .env.dev is local, .env.test is dev, etc.

Reply:
yeah, that is how Wes had configured initially i.e. Dev meant Developer's local env and test is for deployed env on OpenShift. I agree it is confusing. May be we can add a sprint task for this refactoring. I didnt change it, as I am not sure about the impact.